statsd/receiver.go

```diff
@@ -186,14 +197,8 @@ func (mr *metricReceiver) getAdditionalTags(addr string) types.Tags {
 	return nil
 }
 
-// ParseLine with lexer impl.
-func (mr *metricReceiver) parseLine(line []byte) (*types.Metric, error) {
-	llen := len(line)
-	if llen == 0 {
-		return nil, nil
-	}
-	metric := &types.Metric{}
-	metric.Tags = append(metric.Tags, mr.tags...)
```

Comment:
We don't know if it will be metric or event so we add tags in the method above ^
Other things are handled by lexer itself.